src/components/Documents/UploadDocs.tsx

```diff
@@ -161,49 +203,112 @@ class UploadDocs extends React.Component<Props, State> {
-                  <input
-                    type="file"
-                    accept="application/pdf"
-                    id="potentialPdf"
-                    multiple
-                    onChange={this.handleChangeFileUpload}
-                    hidden
-                  />
-                </label>
-                {pdfFiles && pdfFiles.length > 0 ? (
-                  <button type="submit" className={`btn btn-success ld-ext-right ${buttonState}`}>
-                    Upload
-                    <div className="ld ld-ring ld-spin" />
+        <div className="card-alignment mb-3 pt-5">
+          {currentStep === 0 && (
+            <div>
+              <p className="lead pt-3">
+                Select a PDF file to upload.
+                The name of the PDF will appear when loaded.
+                After confirming that you have chosen the correct file, click the &quot;Submit&quot; button to upload.
+                Otherwise, choose a different file.
+              </p>
+              <DropzoneUploader
+                pdfFiles={this.state.pdfFiles}
+                maxNumFiles={MAX_NUM_OF_FILES}
+                currentStep={this.state.currentStep}
+                updateStep={this.updateStep}
+                updateFileList={this.updateFileList}
+              />
+            </div>
+          )}
+          {currentStep === 1 && (
+            <div className="container">
+              <p className="lead pt-3">
+                Please review the documents below. Click the &quot;Continue&quot; button to submit.
+              </p>
+              <Container>
+                {
+                  pdfFiles && pdfFiles.length > 0 ? Array.from(pdfFiles).map((pdfFile, index) =>
+                    (
+                      <div className="container mb-3 card-alignment">
+                        <Card style={{ width: '48rem' }}>
+                          <Row className="row-padding g-4 md-3">
+                              <Col sm={8}>
+                                      <Col sm={10}>
+                                        <Card.Title>{pdfFile.name}</Card.Title>
+                                        <RenderPDF
+                                          key={uuid()}
+                                          pdfFile={pdfFile}
+                                        />
+                                      </Col>
+                              </Col>
+                            <Col sm={4}>
+                              <DropdownButton
+                                title={this.setTitle(index)}
+                              >
+                                <Dropdown.Item eventKey="License" onClick={(event) => this.handleOnClickCard(index, 'Driver License')}>Driver&apos;s License</Dropdown.Item>
+                                <Dropdown.Item href="#" onClick={(event) => this.handleOnClickCard(index, 'Social Security Card')}>Social Security Card</Dropdown.Item>
```

Comment:
just wondering why we have an href attribute on here- is this needed?

Reply:
not needed - I'll remove them